test/test_nn.py

```diff
@@ -13943,6 +13943,9 @@ def helper(n, c, h, w, kernel_size, stride=None,
         helper(4, 8, 8, 8, 3, count_include_pad=False, padding=2, stride=2)
         helper(4, 8, 8, 8, 3, divisor_override=42)
         helper(4, 8, 8, 8, 7)
+        # ROCm 16GB MI25 hits OOM error. Clear caching allocator prior to running large subtest.
+        if 'cuda' in device:
```

Comment:
won't  this also clear the cache line in CUDA builds as well instead of only on ROCm builds? is that a desire behavior also on CUDA?

Reply:
Resolved with 21af4538e85f4d63a288a1c5d1f1ed78440045b4.  Logic only applies to rocm now.